docs/administration/spaces/index.md

```diff
@@ -4,76 +4,134 @@ description: Spaces setup and configuration.
 
-### Create a Space
+### The Space Manager
+
+Each space has a **Space Manager**. The space manager is the Administrator for that Space and is responsible for managing users and teams to the space and assigning permissions to them.
+
+When creating a new space, you are required to nominate a team member (or a team) to the role of space manager. This space manager is then responsible for [managing teams and permissions](/docs/managing-users-and-teams) within that space.
+
+The user who creates a space doesn't necessarily need to be the space manager of the space. This enables a 'hands off' administrative approach suited to larger organizations or those who which to which to separate the duties of Octopus Server Administration, from the duties of Team Administration.
+
+Behind the scenes, a **Space Managers** team is created, and any users that are nominated to be a space manager, are put in that team. This team cannot be created or deleted, and serves no other purpose than applying the correct space manager permissions. 
+
+### Create a Space {#create-a-space}
 
 New spaces are added from the configuration section of the portal.
 
 1. To create a new Space navigate to {{Configuration,Spaces}} and select **ADD SPACE**.
 2. Give the Space a name.
-3. Give the Space an owner. This can be individual users or teams. Either can be selected from the drop-down menu. Click **SAVE**.
+3. Give the Space a manager. This can be individual users or teams. Either can be selected from the drop-down menu. Click **SAVE**.
 4. Provide a description for the Space.
 5. Optionally, upload a logo for the Space.
 6. Click **SAVE**.
 
-### Modify  a Space
+![Add new space](add-new-space.png)
+
+### Modify a Space
 
 You can modify a space by navigating to {{Configuration,Spaces}} and selecting the space you want to modify. Expand the field you would like to change and click **SAVE** to save your changes.
 
+Here you can rename a space, give it a new logo, modify the space managers, or even stop the spaces task queue from processing.
+
+![Modify a space](modify-space.png)
+
 ### Delete a Space
 
-You can delete spaces when you are the **Space Owner**. Deleting a space cannot be undone, and the space and all of its contents, including projects, environments, releases, and deployment history will be deleted.
+You can delete spaces when you are the **Space manager**. Deleting a space cannot be undone, and the space and all of its contents, including projects, environments, releases, and deployment history will be deleted.
 
-1. Navigate to {{Configuration,Spaces}} and selecT the space you want to delete.
+1. Navigate to {{Configuration,Spaces}} and select the space you want to delete.
 1. Expand the **Task Queue Status** section and select the Stop task queue checkbox, and click **SAVE**.
 1. Click the overflow button and select **Delete**.
 1. Enter the name of the space and click **DELETE**.
 
+## Change the Default Space {#change-the-default-space}
```

Comment:
Probably worth mentioning the API in this section - say that the primary effect of the default space option is that any REST API calls where the space is not explicitly specified will be forwarded to the default space.

Reply:
I know you've mentioned this below in the disable the default space section, but I think it should be up in this section.

Reply:
Actually maybe a better way to structure this would be like:

```
## Default space

### Change or disable the default space
```

You can give some background about what the default space is and what it means for the API in the parent section. Changing or disabling the default space are cohesive because they have a similar effect on the API (potentially breaking existing scripts)